src/Domain/Entity/Promotion.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace BBC\ProgrammesPagesService\Domain\Entity;
+
+use BBC\ProgrammesPagesService\Domain\ValueObject\Pid;
+use BBC\ProgrammesPagesService\Domain\ValueObject\Synopses;
+
+class Promotion
+{
+    /** @var bool */
+    private $cascadesToDescendants;
+
+    /** @var Pid */
+    private $pid;
+
+    /** @var PromotableInterface */
+    private $promotedEntity;
+
+    /** @var Synopses */
+    private $synopses;
+
+    /** @var string */
+    private $title;
+
+    /** @var string */
+    private $url;
+
+    /** @var int */
+    private $weighting;
+
+    public function __construct(
+        bool $cascadesToDescendants,
+        Pid $pid,
+        PromotableInterface $promotedEntity,
+        Synopses $synopses,
+        string $title,
+        string $url,
+        int $weighting
+    ) {
+        $this->cascadesToDescendants = $cascadesToDescendants;
+        $this->pid = $pid;
+        $this->promotedEntity = $promotedEntity;
+        $this->synopses = $synopses;
+        $this->title = $title;
+        $this->url = $url;
+        $this->weighting = $weighting;
+    }
+
+    public function isCascadesToDescendants(): bool
```

Comment:
I don't think we'd ever need to call this from code, we only use it for filtering in DB queries, so I don't think there's any value having it here at all.